Editor/Scripts/CodeGenerators/CodeGeneratorUtils.cs

```diff
@@ -0,0 +1,12 @@
+using System.Text.RegularExpressions;
+
+namespace RegressionGames
+{
+    public class CodeGeneratorUtils
+    {
+        public static string SanitizeActionName(string name)
```

Comment:
The same code should also apply to states, I believe? Can we also fix that here, or we can wait until we see where we get with RGState

Reply:
Oh interesting, thanks!